Source/WebCore/page/AutoscrollController.cpp

```diff
@@ -75,6 +77,8 @@ void AutoscrollController::startAutoscrollForSelection(RenderObject* renderer)
     if (m_autoscrollTimer.isActive())
         return;
     auto* scrollable = RenderBox::findAutoscrollable(renderer);
+    if (!scrollable)
+        scrollable = renderer->isListBox() ? downcast<RenderListBox>(renderer) : nullptr;
```

Comment:
Why are we leaving AutoscrollController::updateDragAndDrop in tact?
It seems like we should be fixing both functions.

Reply:
Dropping does not lead to 'text' / label of options being presented in any Notes or other application, so I think 'onChange' on start would be appropriate. Similarly, if we don't have 'dropTargetNode', which if I understand correctly would mean without any 'node', will just early return as 'nullptr', if we do it in 'updateDragAndDrop' function.

Reply:
@rniwa - From Slack 1-1: If I understand function correctly ‘updateDragAndDrop’ needs dropTarget but option text / label does not get dropped and don’t have node so I think it will early return and will not work

Reply:
I see.